packages/leafygreen-provider/src/PopoverPropsContext/PopoverPropsContext.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { act, render, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { renderHook } from '@leafygreen-ui/testing-lib';
+
+import {
+  PopoverPropsProvider,
+  usePopoverPropsContext,
+} from './PopoverPropsContext';
+import { type PopoverPropsProviderProps } from './PopoverPropsContext.types';
```

Comment:
`act`, `waitFor`, `userEvent`, and `PopoverPropsProviderProps` are imported but not used.